packages/runtime/src/widgets/Form/Radio.tsx

```diff
@@ -45,7 +45,7 @@ export const RadioWidget: React.FC<RadioWidgetProps> = (props) => {
   const formInstance = Form.useFormInstance();
 
   useEffect(() => {
-    setValue(String(values?.initialValue));
+    setValue(values?.initialValue);
```

Comment:
@anserwaseem if you remove this type casting here, the initial value, I mean the initial selected radio button will not be selected, you need to click two times to reflect the UI changes.

Please test this also.

Reply:
<img width="1243" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/31651612/aa6ac38d-c6aa-47d7-8275-f5b374930e2c">
@anserwaseem check this, as radio button has assigned initial value but not rendered on UI

Reply:
i missed that, ty.